FBControlCore/Diagnostics/FBBatchLogSearch.m

```diff
@@ -299,8 +299,8 @@ + (NSDictionary *)searchDiagnostics:(NSArray<FBDiagnostic *> *)diagnostics withP
 
 + (NSArray<NSString *> *)search:(FBDiagnosticLogSearch *)search withOptions:(FBBatchLogSearchOptions)options
 {
-  BOOL lines = options & FBBatchLogSearchOptionsFullLines;
-  BOOL first = options & FBBatchLogSearchOptionsFirstMatch;
+  BOOL lines = (options & FBBatchLogSearchOptionsFullLines) ? YES : NO;
```

Comment:
The same as the comment above

Reply:
It was needed to resolve warnings about deprecations.